### PR TITLE
add low pt photons (10-14 GeV) to miniAOD for 76x

### PR DESCRIPTION
**Pull request by cmkuo**
add photons with pt between 10 and 14 GeV with H/E < 0.15 and charged hadron isolation < 10 GeV

### CODE REVIEW

**Comment by lgray** [comment]
@cmkuo Can you please make the title of this PR (and the backport) more descriptive? Thanks!

**Comment by lgray** [comment]
@cmsbuild please test

**Comment by lgray** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
@cmkuo:
I'm having difficulty finding the new low-pT photons in Mini-AOD. Could you please suggest a workflow I can run that will show the new photons getting added to Mini-AOD?

**Comment by cmkuo** [comment]
@cvuosalo Do you mean a physics process or something else ? If it's a physics process, which process did you run ?

**Comment by cvuosalo** [comment]
@cmkuo: Sorry, my mistake. I found the new photons.

**Comment by cvuosalo** [comment]
+1

For #10665 ca442a0f8985c795e69940b8703962894ca926e8

Adding low-pT photons to Mini-AOD. There should be no changes except in Mini-AOD.

The code changes are satisfactory, and Jenkins [tests](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-10665/7133/summary.html) against baseline CMSSW_7_6_X_2015-08-10-2300 show no significant differences, as expected. Checking workflows 1318.0_SingleGammaPt10 and 4.77_ZElSkim2012D+ZElSkim2012D against baseline CMSSW_7_6_0_pre2 shows the reducedEgamma collections in Mini-AOD do increase in size with the additional photons, an increase of 2.6% for 1318 and 0.3% for 4.77.

File: RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
 reducedEgamma = cms.EDProducer("ReducedEGProducer",
-  keepPhotons = cms.string("pt > 14 && hadTowOverEm()<0.15"), #keep in output
-  slimRelinkPhotons = cms.string("pt > 14 && hadTowOverEm()<0.15"), #keep only slimmed SuperCluster plus seed cluster
+  keepPhotons = cms.string("(pt > 14 && hadTowOverEm()<0.15) || (pt>10 && pt<=14 && hadTowOverEm()<0.15 && chargedHadronIso()<10)"), #keep in output
```
Comment (davidlange6) [inline RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py]: @cmkuo 
a much simpler logic that does the same thing would be 
hadTowOverEm()<0.15 && pt>10 && ( pt>14 || chargedHadronIso()<10)

**Comment by lgray** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
+1

For #10665 ce17dd81326c24a8e6bc6b646b480dfc2226c266

Second approval after code revision.

The code changes are satisfactory, and Jenkins tests against baseline CMSSW_7_6_X_2015-08-13-2300 show no significant differences, as expected. Checking workflow1318.0_SingleGammaPt10 against baseline CMSSW_7_6_0_pre2 shows the reducedEgamma collections in Mini-AOD do increase in size by 2.6% with the additional photons.

**Comment by davidlange6** [comment]
+1

**Pull request merged**